controllers/service_controller.go

```diff
@@ -48,32 +47,39 @@ const (
 
 // ServiceReconciler reconciles a Service object
 type ServiceReconciler struct {
-	client.Client
+	log              gwlog.Logger
+	Client           client.Client
 	Scheme           *runtime.Scheme
 	finalizerManager k8s.FinalizerManager
 	eventRecorder    record.EventRecorder
 	modelBuilder     gateway.LatticeTargetsBuilder
 	stackDeployer    deploy.StackDeployer
 
-	latticeDataStore *latticestore.LatticeDataStore
```

Comment:
Why make this less readable?

Reply:
looks fine to me, but can return it back